### DISCUSSION TITLE
wishlist/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#430] [Website Discussions]
# wishlist/

This page is dedicated to keeping track of Final Cut Pro feature requests, via our GitHub issues site

https://fcp.cafe/wishlist/

**Comment by JohnTheHomeSapien** [comment]
Be able to flatten Multicam Clips on time line

**Comment by Ado-prog** [comment]
Subject: Add "Set Poster Frame" option to FCP Share/Export menu

Final Cut Pro automatically picks the thumbnail frame based on playhead position during export, but it's unreliable for Finder thumbnails and TV/USB playback. 

Please add a "Set Poster Frame" button directly in the Share menu (like the marker in Compressor) so we can choose the exact frame for the video's thumbnail. Currently requires extra Compressor step. 

Many users request this (Apple forums, Reddit, FCPXUsers FB group).
